### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ombralunare**
What I did so far ...

### CODE REVIEW

**Review by CharlSteynberg — CHANGES_REQUESTED** [review]

File: index.html [hunk 1]
```diff
@@ -4,9 +4,11 @@
         <title>Ricordami</title>
         <link rel="stylesheet" href="style.css"></link>
         <link rel="stylesheet" href="icomoon.css"></link>
+        <link rel="audio" href="audio.mp"></link>
```
[the remark this answers is not in the document]
Comment (ombralunare) [reply index.html:7]: done

File: script.js [hunk 2]
```diff
@@ -121,3 +119,14 @@
 
 
     refresh();
+
+
+//createElementbyID()   - audio href?
+
+var audioElement = new Audio(url);  // creates audio element for alarm //
```
[the remark this answers is not in the document]
Comment (ombralunare) [reply script.js:126]: please let me know how this is looking so far...
(haven't tested, waiting for ur approval/Review)

I haven't done the cards yet. Not too sure how to go about it. (is it similar to the code for "cardName"??) 
I haven't done any html yet - must i make the alarm button (loudButn) a 'submit' attr??

**Review by CharlSteynberg — COMMENTED** [review]
have you tested this?

**Comment by ombralunare** [comment]
I've tested it and it doesn't work. It doesn't store the created card or provide a new one... 

Will get on it

**Pull request merged**